tests/server.py

```diff
@@ -73,11 +73,13 @@
 from synapse.config.database import DatabaseConnectionConfig
 from synapse.config.homeserver import HomeServerConfig
 from synapse.events.presence_router import load_legacy_presence_router
-from synapse.events.third_party_rules import load_legacy_third_party_event_rules
 from synapse.handlers.auth import load_legacy_password_auth_providers
 from synapse.http.site import SynapseRequest
 from synapse.logging.context import ContextResourceUsage
 from synapse.module_api.callbacks.spamchecker_callbacks import load_legacy_spam_checkers
+from synapse.module_api.callbacks.third_party_event_rules_callbacks import (
+    load_legacy_third_party_event_rules,
+)
```

Comment:
Any plans to remove legacy stuff? 👼

Reply:
Yes... I will get back to this. There's a few to eliminate:

https://github.com/matrix-org/synapse/blob/a924433e24e2ac81ca69665d8c737c5754b6c734/tests/server.py#L1053-L1056